docs/content/reference/static-configuration/env-ref.md

```diff
@@ -87,6 +87,12 @@ HTTP challenge EntryPoint
 `TRAEFIK_CERTIFICATESRESOLVERS_<NAME>_ACME_KEYTYPE`:  
 KeyType used for generating certificate private key. Allow value 'EC256', 'EC384', 'RSA2048', 'RSA4096', 'RSA8192'. (Default: ```RSA4096```)
 
+`TRAEFIK_CERTIFICATESRESOLVERS_<NAME>_ACME_RENEWBEFOREEXPIRY`:
+Time remaining before the certificate expiration when Traefik will try to renew it. (Default: ```720h0m0s```)
+
+`TRAEFIK_CERTIFICATESRESOLVERS_<NAME>_ACME_EXPIRATIONCHECKINTERVAL`:
+Frequency in which Traefik will check if certificate is due for renewal. (Default: ```24h0m0s```)
+
```

Comment:
This file has a header that says:

CODE GENERATED AUTOMATICALLY
THIS FILE MUST NOT BE EDITED BY HAND

So I'm removing this change.

Reply:
This file must be generated and committed in your PR.

```console
$ go generate
```